src/helix-ui.less

```diff
@@ -1,138 +1,8 @@
-/* ========== CONFIGURATION ========== *\
-  - @import
-  - LESS/CSS Variables
-  - LESS Mixins
-  - NO Element Selectors
-  - NO Pseudo Selectors
-  - NO !important
-\* ========== CONFIGURATION ========== */
-@import url("https://fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,400italic,500,500italic,700,700italic");
-@import url("https://fonts.googleapis.com/css?family=Roboto+Mono:400,100,100italic,300,300italic,400italic,700,700italic");
-@import (reference) 'vars';
-
-@keyframes hx-spin {
-  0% { transform: rotate(0deg); }
-  100% { transform: rotate(360deg); }
-}
-
-
-/* ========== RESETS ========== *\
-  - Element & Pseudo Selectors
-  - No Class Selectors
-  - NO !important
-\* ========== RESETS ========== */
+// DO NOT REORDER!!!
+@import 'config';
 @import 'reset';
-@import 'reset/html5';
-@import 'reset/table';
-@import 'reset/typography';
-
-
-/* ========== BASE ========== *\
-  - Elements Selectors ONLY
-  - No Class Selectors
-  - NO !important
-\* ========================== */
-@import 'base/links';
-@import 'base/lists';
-@import 'base/typography';
-
-// TODO: update this once spacing system is defined
-figure {
-  margin-bottom: 1rem;
-}
-
-// Core CSS
-@import 'core/hx-accordion';
-@import 'core/hx-accordion-panel';
-@import 'core/hx-alert';
-@import 'core/hx-busy';
-@import 'core/hx-checkbox';
-@import 'core/hx-dd';
-@import 'core/hx-def';
-@import 'core/hx-disclosure';
-@import 'core/hx-dl';
-@import 'core/hx-error';
-@import 'core/hx-icon';
-@import 'core/hx-menu';
-@import 'core/hx-menuitem';
-@import 'core/hx-menuitem-group';
-@import 'core/hx-modal';
-@import 'core/hx-modalbody';
-@import 'core/hx-modalfoot';
-@import 'core/hx-modalhead';
-@import 'core/hx-panel';
-@import 'core/hx-panel-body';
-@import 'core/hx-panel-foot';
-@import 'core/hx-panel-head';
-@import 'core/hx-popover';
-@import 'core/hx-popover-body';
-@import 'core/hx-popover-foot';
-@import 'core/hx-popover-head';
-@import 'core/hx-reveal';
-@import 'core/hx-search';
-@import 'core/hx-search-assistance';
-@import 'core/hx-status';
-@import 'core/hx-tab';
-@import 'core/hx-tabcontent';
-@import 'core/hx-tablist';
-@import 'core/hx-tabpanel';
-@import 'core/hx-tabset';
-@import 'core/hx-tile';
-@import 'core/hx-tile-description';
-@import 'core/hx-tile-title';
-@import 'core/hx-tooltip';
-
-
-/* ========== COMPONENTS ========== *\
-    - Complex CSS Selectors
-    - NO !important
-\* ========== COMPONENTS ========== */
-@import 'components/alerts';
-@import 'components/layouts';
-@import 'components/search';
-
-// MODULES (TODO: convert to components)
-@import 'modules/box';
-@import 'modules/breadcrumb';
-@import 'modules/button';
-@import 'modules/choice-tiles';
-@import 'modules/grid';
-@import 'modules/hljs';
-@import 'modules/pagination';
-@import 'modules/stepper'; // BETA
-@import 'modules/table';
-@import 'modules/text-input';
-@import 'modules/typography';
```

Comment:
`modules/*` imports moved to `components.less` and renamed to match their name in the component explorer